src/components/Router/StepComponentMap.js

```diff
@@ -31,6 +33,25 @@ const createDocumentComponents = (documentType) => {
   return frontDocumentFlow
 }
 
+
+const crossDeviceComponents = {
+  mobileLink: () => [MobileLink],
+  mobileConnection: () => [MobileFlowInProgress, MobileFlowComplete]
```

Comment:
This should be one component rather than two right? It's fine if you plan to fix as part of CX-1148

Reply:
Yes, it's outside the scope of this ticket, I've only added the components to test the navigation